server/controllers/index.js

```diff
@@ -1,8 +1,11 @@
 // TODO: FlowType this module
 import {type Promo} from '../model/promo';
+import {wellcomeCollection} from '../model/organization';
 import {createPageConfig} from '../model/page-config';
 import {getPosts, getArticle} from '../services/wordpress';
 
+export const site = wellcomeCollection;
```

Comment:
[We already add `organization` to the pageConfig](https://github.com/wellcometrust/wellcomecollection.org/blob/master/server/model/page-config.js#L16)